package.json

```diff
@@ -35,8 +32,10 @@
       "last 1 safari version"
     ]
   },
-  "resolutions": {
-    "eslint": "^8.0.0"
+  "overrides": {
+    "vite": {
+      "rollup": "npm:@rollup/wasm-node"
+    }
```

Comment:
Why is this needed? Is this related to the problems of the docker container failing on build?

The reason why this is needed could be documented with a sort of a comment (I mean JSON format doesn't have comments, but they can be sort of emulated with "//" key) in package.json, like e.g. in palvelutarjotin-admin:
- https://github.com/City-of-Helsinki/palvelutarjotin-admin/blob/palvelutarjotin-admin-v1.14.0/package.json#L13-L24